.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,27 +1,60 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-title: ''
+title: "🐛 "
```

Comment:
Looks like there is a space after the emoji, is that right or am I seeing things

Reply:
yes, this is intentional. The user can type after the emoji right away without needing to press space